src/vs/workbench/contrib/notebook/common/notebookEditorModel.ts

```diff
@@ -309,7 +309,7 @@ export class NotebookFileWorkingCopyModel extends Disposable implements IStoredF
 	}
 
 	pushStackElement(): void {
-		this._notebookModel.pushStackElement('save', undefined, undefined);
+		this._notebookModel.pushStackElement();
```

Comment:
Save requires pushing current stack element into unto redo (i.e. we need to close it)
No more `save` operations.